source/index.html.md.erb

```diff
@@ -0,0 +1,44 @@
+---
+title: Tito API Documentation
+
+language_tabs: # must be one of https://git.io/vQNgJ
+  - shell
+
+toc_footers:
+  -
+
+includes:
+
+search: true
+---
+
+# Introduction
+
+> **API endpoint**
+
+```
+<%= config[:api_endpoint] %>
+```
+
+> Examples in this documentation are written using curl.
+
+The Tito API is designed to be a predictable and intuitive interface for secure programmatic access to Tito.
+
+The Tito API is a REST API and returns JSON responses.
+
+We hope by providing an API it will allow developers to build cool and useful tools on top of Tito.
+
+# Authentication
+
+```shell
+curl --request GET \
+  --url '<%= config[:api_endpoint] %>...' \
+  --header 'Authorization: Token token=YOUR-API-KEY' \
+  --header 'Accept: application/json' \
+```
+
+The Tito API uses API keys to allow users to authenticate without exposing their credentials.
```

Comment:
I am wondering if for consistency we should call this a token. In ID tito we call it a token. That also matches the authorization header name. There is nothing worse as an API developer wonder if you are using the right token/key so probably better to be consistent? Thoughts?

Reply:
Decided to go ahead and change to API token :

2c086f313e28ae0f6eb01ff2bcc95c759063e4a2